docs/release_notes/8.0.1-1.md

```diff
@@ -0,0 +1,83 @@
+
+#### Scaling improvements
+
+* To better optimize resources or distribute the data more evenly across the shards, you can now [move a single unsharded collection to any shard](https://www.mongodb.com/docs/manual/core/moveable-collections/#std-label-moveable-collections) with the new `moveCollection` command. This operation is going online and the collection data is fully available during the migration. This new way to move data is flexible and the read and write behavior when you query data is transparent.
+* Ensuring even data distribution across shards in your cluster is crucial. Starting with MongoDB 5.0, you can reshard a collection using a different shard key, although this operation is time-consuming. In MongoDB 8.0, you can reshard a collection using the same shard key, significantly boosting performance. This is especially beneficial for large collections, reducing resharding time from days to several hours. You can dynamically reallocate your resources based on the workload. The process is online, so your operations run uninterrupted. The ability to add or remove shards without changing the shard key makes the whole resharding process straightforward. 
+* You can now [unshard a collection](https://www.mongodb.com/docs/manual/tutorial/unshard-collection/#std-label-unshard-collection-task) if it should reside on a single shard or the collection requires resource isolation. You can either specify the desired shard or the collection will be moved onto the shard with the least load. 
+* You can now save costs on additional computing resources when deploying a sharded cluster. The config server replica set can now store both the configuration and the database data. Such a deployment is recommended for small clusters and is especially useful for evaluating the cluster without having to pay extra costs.
+* You can now make self-managed backups in sharded clusters using `mongodump`. Run the `fsync` command with the `lock` field set to `true` from `mongos` to lock each shard and prevent further writes. Then, run `fsynckUnlock` to unlock shards.
+
+#### Logging improvements
+
+You can now configure database profiler to log slow operations based on the time MongoDB actually spent processing that operation. This gives you more accuracy when you analyze performance. 
+
+#### Security improvements
+
+* You can now run range queries on encrypted fields, specifically for numeric data types and dates. This expands the range of use cases for queryable encryption, including accounting operations, healthcare, financial services, and other strictly regulated data operations where data safety is crucial.
```

Comment:
```suggestion
```
it's enterprise

Reply:
Manual Queryable encryption is in Community so I think this improvement applies to us as well. https://www.mongodb.com/docs/manual/core/queryable-encryption/reference/compatibility/#queryable-encryption-compatibility

Reply:
So do we keep it or delete it?

Reply:
I think we keep it. Please confirm @sandraromanchenko

Reply:
@radoslawszulgo true, but the mentioned improvement AFAIU is related to automatic encryption
https://www.mongodb.com/docs/manual/core/queryable-encryption/reference/supported-operations/#std-label-qe-reference-automatic-encryption-supported-operations

Reply:
I understand it differently: 
"It outlines the behavior for operations using automatic encryption, and operations using explicit encryption."

Reply:
Explicitly stated Manual Queryable encryption